opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/SingleNumericalAquifer.hpp

```diff
@@ -57,7 +57,9 @@ namespace Opm {
 
         std::unordered_map<size_t, AquiferCellProps> aquiferCellProps() const;
 
-        std::vector<NNCdata> aquiferNNCs(const EclipseGrid& grid, const FieldPropsManager& fp) const;
+        std::vector<NNCdata> aquiferCellNNCs() const;
+        std::vector<NNCdata> aquiferConnectionNNCs(const EclipseGrid &grid, const FieldPropsManager &fp) const;
```

Comment:
FYI. For NNCs from numerical aquifers, it can be between the aquifer cells, and also between aquifer cell and aquifer connections. For the one between aquifer cells, we can generate on-site, for the one between aquifer cell and aquifer connections, we do it during the grid processing in opm-grid. That is why we split the `aquifernNNCs` to be two separate functions here.